src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    public static Task[] tasks = new Task[100];
```

Comment:
I really like how you named the array in its plural form of "tasks". It sounds more intuitive